### ISSUE TITLE
Change textfield border color 

### ISSUE THREAD

**Issue by hv14** [hv14/RP-Login-Page#3]
After a user types in a correct input into the username and password textfields, the border color should be blue

**Comment by likuilin** [comment]
What format defines a correct input? Anything other than nothing?

**Comment by hv14** [comment]
Yep for now as long as the user has inputted something it's correct input

**Comment by likuilin** [comment]
I don't think that's possible in pure html+css... maybe some finagling with placeholder text styling might work, but that's way too hacky - is adding javascript okay?

**Comment by pranaygp** [comment]
@likuilin the HTML spec actually does support a "minlength" attribute to verify inputs (https://www.w3.org/TR/html5/forms.html#the-maxlength-and-minlength-attributes). If you want to do more advanced verification, the spec also supports "pattern" (https://www.w3.org/TR/html5/forms.html#the-pattern-attribute) to verify inputs. Technically, you can do this without JS.

However, if you choose to use those, I believe you don't get fine control over valid/invalid styling and are stuck to browser defaults. JS would definitely let you exercise more control. 

I think either solution is fine :)
